### PR TITLE
Hide window

### LINKED ISSUE #25794: In immersive mode, make sure to mark the window as not visible
I haven't tested yet, but we should make sure, at least in the case of the HL, that the compositor pipeline is set to not visible, to avoid useless animation to run in the background.

### PR DESCRIPTION
**Pull request by paulrouget**
Stop animations when the window is hidden.

Tested with both Immersive Mode, and regular window being minimized.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #25794 (GitHub issue number if applicable)

### CODE REVIEW

File: support/hololens/ServoApp/BrowserPage.cpp [hunk 1]
```diff
@@ -71,6 +71,10 @@ void BrowserPage::BindServoEvents() {
                 ? Visibility::Collapsed
                 : Visibility::Visible);
       });
+  Window::Current().VisibilityChanged(
```
Comment (Manishearth) [inline support/hololens/ServoApp/BrowserPage.cpp]: I assume this gets called when we enter immersive mode on the hololens as well, right? Is there an FPS improvement?

Comment (paulrouget) [reply support/hololens/ServoApp/BrowserPage.cpp]: > I assume this gets called when we enter immersive mode on the hololens as well, right?

Yes.

> Is there an FPS improvement?

I haven't looked.

Comment (paulrouget) [reply support/hololens/ServoApp/BrowserPage.cpp]: I don't see obvious improvements, but the emulator is probably not the best way of measure that.

**Comment by jdm** [comment]
@bors-servo retry
* #24726 #25513

**Comment by jdm** [comment]
@bors-servo retry
* #24726

**Pull request merged**